### PR TITLE
restores stunning slips

### PR DESCRIPTION
**Pull request by ghost**
restores slips to its fullest so that clown can finally removes shoes off the captain. again.
first pr basically stole off hippies revert 
crawling was a mistake

why: it's a iconic part of the game that resonates with clown and assistant players everywhere, removing it hurt me and i am currently in a great depression due to the lack of it (sympathy pls)

lemme know if i fucked up some of it

noteworthy: punches dont stun anymore and dont know where punch is located, other than that everything is done

### CODE REVIEW

**Comment by Nickvr628** [comment]
I agree. The nerf was dumb and made slips basically useless since the captain can still baton/shoot you the moment he gets slipped.

**Comment by 4dplanner** [comment]
I mean where you leave `//knockdown_time = max(_knockdown, 0)` in the code, but non-functional

**Comment by 4dplanner** [comment]
Further, it's pretty confusing that now _knockdown sets the paralyse time and _paralyze does nothing

**Comment by duckay** [comment]
well i mean i never touched this code before but i tested it and it works so thats good enough for a newbie

**Comment by HydroFloric** [comment]
While I don't think reverting stuns is the way to go. I don't currently like how bad they currently are. 

If  anything I would like to see the stun readded to certain slips such as 100 pot bananas, syndie soaps and lube and either readd the item throw on slip or add a small window in which you cant pick something right back up after slipping. Also possibly locking item removal from bags while on the ground. 

This way slips are no longer a instant you win move but still are powerful in combat.

File: code/datums/components/slippery.dm [hunk 1]
```diff
@@ -5,9 +5,8 @@
 	var/lube_flags
 	var/datum/callback/callback
 
-/datum/component/slippery/Initialize(_knockdown, _lube_flags = NONE, datum/callback/_callback, _paralyze, _force_drop = FALSE)
-	knockdown_time = max(_knockdown, 0)
-	paralyze_time = max(_paralyze, 0)
+/datum/component/slippery/Initialize(_knockdown, _lube_flags = NONE, datum/callback/_callback, _paralyze, _force_drop = TRUE)
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/components/slippery.dm]: If you're not using _knockdown why is the argument kept in?

**Comment by ExcessiveUseOfCobblestone** [comment]
I don't mind a minute (small, not 60 seconds lol) stun just so you can have a very brief window to flee/follow up, but 0 stun and old stun are equally loltier.

**Comment by duckay** [comment]
Whole heartedly disagree, you have galoshes, mag and adv to avoid these "paladins of combat" and your own eyes to deal with them and I yet have to see a slip complaint from the other servers to tell me it is a obstruction to fun combat

**Comment by duckay** [comment]
Not at my desk to fix it right now

**Comment by evsey9** [comment]
how about you make it so that it stuns only when you're running, like with water? But still would knockdown if walking on a banana.

**Comment by duckay** [comment]
Too god damn tired for this

**Comment by 4dplanner** [comment]
The problem is in victim.slip

**Comment by 4dplanner** [comment]
which takes knockdown as its first argument

**Comment by subject217** [comment]
i ded plz buff my iconic slips

**Comment by duckay** [comment]
unironically ided

**Comment by subject217** [comment]
for real though you have a far better chance of getting a mild stun or something merged than you do a full revert

one idea I had was to make people immobilized when you grabbed them while they were crawling - like soft crit, and you'd have to resist out of it.

**Comment by duckay** [comment]
thats something i can't understand, most people i've met hated the nerf and i felt like it was the right thing to revert it and hippie has jokes that i don't find funny so i dont go there

i see no problem in reverting it and using soap as a combat tool has been a tradition for literally everyone involved and now we barely see slips outside fire extinguishers because all slips serve the same god damn purpose unlike before-- we've been with slips for years and even nerfs such as lesser time on the floor but even that didnt even please the unrobust

**Comment by ExcessiveUseOfCobblestone** [comment]
The "initial short stun followed by staying crawled" is a sane compromise, asking for a full revert is kneejerk at best and is very unlikely to get merged.

The whole point of the rework is for stuns to be an initiator rather than an end-all.

**Comment by duckay** [comment]
A unwanted rework at best.
I also mentioned that no-slips and advanced magnetic boots exist within the game which traitors often use in combat, and I'd like to note that people have been fine with slips being slips for ages and suddenly everyone has a problem with them after kevin messes with it.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
This isn't correct. See #41068 for how to revert this properly.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm pretty sure people have been complaining about stuns for literal years, and slips in the past couple particularly.

To imply oranges just decided that he hates stuns so nerf nerf nerf out of nowhere btfo players really just reinforces the idea this is kneejerk.

**Comment by duckay** [comment]
i didnt even mention oranges and i'm not blaming him at all for anything, i just feel like its justified to bring slips the way its meant to be

**Comment by subject217** [comment]
i'm sure the assistant mains are livid that their game uniqueness is ruined, it doesn't mean that the change was actually bad overall.

**Comment by anconfuzedrock** [comment]
it was a bad change but that's because oranges probably isn't done working on stuns and slips. We should just give him some more time for the time being. I think this is just going to waste everyone's time.

**Comment by duckay** [comment]
in all fairness if he was working on something he'd say it already though nobody has asked

**Comment by duckay** [comment]
clown main actually and I think too many are focused on combat so that clown cannot whip the nae nae on other people ruining his most favourable part. To add to this the majority did not want this nerf merged and it shows both in this and the pr nerfing slips directly. I'm not going to let slips allow people to not get punished for not looking where they were going. You aren't going to intimidate me like that.

**Comment by optimumtact** [comment]
good post

**Comment by optimumtact** [comment]
just so we're absolutely clear, anybody who touches these values and doesn't run it by me first is going to get their shit pushed in repeatedly.

**Comment by duckay** [comment]
you can push my shit in anytime

**Comment by duckay** [comment]
do you have plans for slips tho

**Comment by optimumtact** [comment]
yeah, it's to leave them nerfed because the meta of the fleeing party always having the advantage is the wrong one.

**Comment by duckay** [comment]
You guys have eyes don't you? 
https://i.imgur.com/ugfWtjB.png

**Comment by duckay** [comment]
Here's a better idea how about buffing clown pda, syndi soap, ice(?) and hi potency bananas to keep things the way they were or nerf stun time even more. Just a thought

**Comment by GuyonBroadway** [comment]
@optimumtact

Git good lmao!

**Pull request closed without merging**